updates/6/updates.sql

```diff
@@ -455,4 +455,20 @@ INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xc9BC48c72154e
 INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xF256CC7847E919FAc9B808cC216cAc87CCF2f47a", 18, "aave-v2"); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xF256CC7847E919FAc9B808cC216cAc87CCF2f47a", "C", "Aave interest bearing XSUSHI", "aXSUSHI", 1613338009, NULL, "xsushi", "", "0xF256CC7847E919FAc9B808cC216cAc87CCF2f47a");
 *
 INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xAa6E8127831c9DE45ae56bB1b0d4D4Da6e5665BD", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xAa6E8127831c9DE45ae56bB1b0d4D4Da6e5665BD", "C", "ETH 2x Flexible Leverage Index", "ETH2x-FLI", 1615709597, NULL, "eth-2x-flexible-leverage-index", "ETH2X-FLI", "0xAa6E8127831c9DE45ae56bB1b0d4D4Da6e5665BD");
-*
\ No newline at end of file
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x824a50dF33AC1B41Afc52f4194E2e8356C17C3aC", 10, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x824a50dF33AC1B41Afc52f4194E2e8356C17C3aC", "C", "KickToken", "KICK", 1624876630, NULL, "kick", "KICK", "0x824a50dF33AC1B41Afc52f4194E2e8356C17C3aC");
+*
+INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("NRV", "S", "Nerve Finance", "NRV", 1614902617, NULL, "nerve-finance", "", "NRV"); INSERT INTO common_asset_details(asset_id, forked) VALUES("NRV", NULL);
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x9695e0114e12C0d3A3636fAb5A18e6b737529023", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x9695e0114e12C0d3A3636fAb5A18e6b737529023", "C", "DFYN Token", "DFYN", 1620371946, NULL, "dfyn-network", "DFYN", "0x9695e0114e12C0d3A3636fAb5A18e6b737529023");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x2fdF40C484b1BD6F1c214ACAC737FEDc8b03E5a8", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x2fdF40C484b1BD6F1c214ACAC737FEDc8b03E5a8", "C", "MCS", "MCS", 1623309370, NULL, "yggdrash", "YEED", "0x2fdF40C484b1BD6F1c214ACAC737FEDc8b03E5a8");
+*
+UPDATE assets SET swapped_for="_ceth_0x2fdF40C484b1BD6F1c214ACAC737FEDc8b03E5a8", cryptocompare="YEED" WHERE identifier="_ceth_0xcA2796F9F61dc7b238Aab043971e49c6164DF375";/*Introduce YEED token swap*/
```

Comment:
Hmm are you sure about this?

Coingecko shows https://www.coingecko.com/en/coins/yggdrash 0xca2796f9f61dc7b238aab043971e49c6164df375 as the address of the token.

Reply:
I got the information from here https://www.bitfinex.com/posts/669

Reply:
and the announcement here https://medium.com/mcs-token-official/introducing-mcs-token-c89f9d50e08b

Reply:
I see. But then they should release a new coingecko entry as this will misclassify the MCS with the YGG icon.

Reply:
Anyway this can be done later when possible.